tests/scripts/all.sh

```diff
@@ -483,16 +483,15 @@ component_test_default_cmake_gcc_asan () {
     msg "test: ssl-opt.sh (ASan build)" # ~ 1 min
     if_build_succeeded tests/ssl-opt.sh
 
-    msg "test/build: ref-configs (ASan build)" # ~ 6 min 20s
-    record_status tests/scripts/test-ref-configs.pl
-
-    msg "build: with ASan (rebuild after ref-configs)" # ~ 1 min
-    make
-
     msg "test: compat.sh (ASan build)" # ~ 6 min
     if_build_succeeded tests/compat.sh
 }
 
+component_test_ref_configs () {
+    msg "test/build: ref-configs (ASan build)" # ~ 6 min 20s
```

Comment:
The `(ASan build)` part of the message is no longer correct. Please either remove it from the message, or add the following line before running the script:
```
CC=gcc cmake -D CMAKE_BUILD_TYPE:String=Asan .
```

Reply:
Fixed by amending "Move test-ref-configs into its own component"